build.sbt

```diff
@@ -68,6 +68,11 @@ lazy val root = project.withId("spark-snowflake").in(file("."))
       // "org.apache.spark" %% "spark-hive" % testSparkVersion % "provided, test"
     ),
 
+    assemblyMergeStrategy in assembly := {
```

Comment:
For my learning purpose, what's assemblyMergeStrategy for?

Reply:
This resolves dependencies when there are conflicts in versions during the build process. This can occur when the same dependency is cached in different places; this strategy AFAIK simply discards one of them and uses the other.

Reply:
It will be great if you can add comment for this. It doesn't block your current PR. You can do in next PR.

Reply:
Done.